CMakeLists.txt

```diff
@@ -29,8 +29,16 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 set(CMAKE_PREFIX_PATH "$ENV{OPENSCAD_LIBRARIES}")
 
 find_package(Eigen3 REQUIRED QUIET)
-message(STATUS "Eigen3: ${EIGEN3_VERSION_STRING}")
-include_directories(${EIGEN3_INCLUDE_DIRS})
+if(DEFINED EIGEN3_VERSION_STRING)
```

Comment:
Perhaps document that this originates from Eigen3Config.cmake, and that `EIGEN3_VERSION` originated from Eigen's own FindEigen3.cmake ?